software/profiling/Vampir.rst

```diff
@@ -1,11 +1,320 @@
 .. _vampir:
 
+.. image:: /images/vampir_logo.png
+   :align: left
+   :width: 230px
+   :height: 130px
+
+|
+|
+|
+|
+|
+|
+
 ************************************
-Vampir (Coming Soon!!)
+Vampir
 ************************************
 
-Vampir is a software performance visualizer focused on highly parallel applications. It presents a unified view on an application run including the use of programming paradigms like MPI, OpenMP, PThreads, CUDA, OpenCL and OpenACC. It also incorporates file I/O, hardware performance counters and other performance data sources. Various interactive displays offer detailed insight into the performance behavior of the analyzed application. Vampir's scalable analysis server and visualization engine enable interactive navigation of large amounts of performance data. Score-P and TAU generate OTF2 trace files for Vampir to visualize.
+Overview
+=========
+
+Vampir is a software performance visualizer focused on highly parallel applications. It presents a unified view on an application run
+including the use of programming paradigms like MPI, OpenMP, PThreads, CUDA, OpenCL and OpenACC.
+It also incorporates file I/O, hardware performance counters and other performance data sources.
+Various interactive displays offer detailed insight into the performance behavior of the analyzed application.
+Vampir's scalable analysis server and visualization engine enable interactive navigation of large amounts of performance data.
+Score-P and TAU generate OTF2 trace files for Vampir to visualize.
+
+
+Usage
+=======
+
+Vampir can be run a few different ways depending on a couple of factors. If you have a large trace file it
+would benefit from utilizing VampirServer to process the trace file with the systems compute power
+while reverse connected to the Vampir GUI. If the trace file is small enough, it would do just fine viewing on a
+login node or your local machine.
+
+.. _vamps:
+
+``Vampirserver`` is the backend software component that can run across multiple compute nodes taking advantage
+of the machines memory, this in turn provides a peak performance for viewing large trace files.
+
+.. Note:: Vampirserver does not take advantage of GPU components
+
+
+This page will go through the 3 different types of connections available for using Vampir on Summit.
+
+For logging onto Summit you will need to specify ``-X`` as part of your ``SSH`` command, this will
```

Comment:
Here, `SSH` should be `ssh` unless you take it out of "code format". 
Also, this paragraph should be restructured a bit. Maybe something like:

The following sections will cover the 3 different methods for using Vampir on Summit. For each method, you will need to enable X11 forwarding when logging in to Summit to allow launching a GUI from Summit. To do so, you can use the SSH option ``-X`` as shown below

```
$ ssh -X <username>@summit.olcf.ornl.gov
``` 

Please visit this link if you need more information for logging onto Summit.